index.js

```diff
@@ -2,12 +2,18 @@
 'use strict';
 
 var debug = require('debug')('ember-css-modules:addon');
+
+var HtmlbarsPlugin = require('./lib/htmlbars-plugin');
 var ModulesPreprocessor = require('./lib/modules-preprocessor');
 var OutputStylesPreprocessor = require('./lib/output-styles-preprocessor');
 
 module.exports = {
   name: 'ember-css-modules',
 
+  isDevelopingAddon: function() {
```

Comment:
Note that this should generally be removed before publishing (I assume it's mostly being used now for debug ability).